genesis/model/contract/smart_contract_test.go

```diff
@@ -272,20 +336,24 @@ func TestSmartContract_GetComposite(t *testing.T) {
 	parent := &mockParent{}
 	sc := NewBaseSmartContract(parent)
 	compositeFactory := BaseCompositeFactory{}
-	composite, _ := sc.CreateComposite(&compositeFactory)
+	composite, err := sc.CreateComposite(&compositeFactory)
 
-	res, err := sc.GetComposite(composite.GetInterfaceKey())
+	assert.NoError(t, err)
+
+	res, err := sc.GetComposite(composite.GetInterfaceKey(), composite.(object.Object).GetClassID())
```

Comment:
May be its better to use compositeFactory instead of composite? As we do in GetOrCreateComposite func